suite-native/module-send/src/hooks/useSendForm.ts

```diff
@@ -0,0 +1,304 @@
+import { useCallback, useEffect, useMemo, useState } from 'react';
+import { useWatch } from 'react-hook-form';
+import { Keyboard } from 'react-native';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { D, pipe } from '@mobily/ts-belt';
+import { useNavigation } from '@react-navigation/native';
+import { isFulfilled } from '@reduxjs/toolkit';
+
+import { getDisplaySymbol, getNetwork } from '@suite-common/wallet-config';
+import {
+    AccountsRootState,
+    FeesRootState,
+    SendRootState,
+    WalletSettingsRootState,
+    composeSendFormTransactionFeeLevelsThunk,
+    selectAccountByKey,
+    selectDeviceUnavailableCapabilities,
+    selectIsAmountInSats,
+    selectNetworkFeeInfo,
+    selectSendFormDraftByKey,
+    sendFormActions,
+    updateFeeInfoThunk,
+} from '@suite-common/wallet-core';
+import { TokenAddress } from '@suite-common/wallet-types';
+import { amountToSmallestUnit, getExcludedUtxos } from '@suite-common/wallet-utils';
+import { useForm } from '@suite-native/forms';
+import {
+    SendStackParamList,
+    SendStackRoutes,
+    StackNavigationProps,
+} from '@suite-native/navigation';
+import { TokensRootState, selectAccountTokenInfo } from '@suite-native/tokens';
+import { useDebounce } from '@trezor/react-utils';
+
+import { useUtxoSelection } from './useUxtoSelection';
+import { useSubscribeForSolanaBlockUpdates } from '../hooks/useSubscribeForSolanaBlockUpdates';
+import { storeFeeLevels } from '../sendFormSlice';
+import { calculateFeeLevelsMaxAmountThunk } from '../sendFormThunks';
+import { SendOutputsFormValues, sendOutputsFormValidationSchema } from '../sendOutputsFormSchema';
+import { FeeLevelsMaxAmount } from '../types';
+import { constructFormDraft } from '../utils';
+
+const getDefaultValues = ({
+    tokenContract,
+    isDestinationTagEnabled,
+}: {
+    tokenContract?: TokenAddress;
+    isDestinationTagEnabled: boolean;
+}): Readonly<SendOutputsFormValues> =>
+    ({
+        isDestinationTagEnabled,
+        outputs: [
+            {
+                amount: '',
+                address: '',
+                fiat: '',
+                token: tokenContract ?? null,
+            },
+        ],
+    }) as const;
+
+export function useSendForm(
```

Comment:
nit: we use arrow functions everywhere possible.
```suggestion
const useSendForm = (...) => (
```

Reply:
eb7d7236c9ebd99824ba4ea015ac82026fd06dd7